sno/repo.py

```diff
@@ -265,23 +266,27 @@ def _create_with_git_command(cls, cmd, gitdir_path, temp_workdir_path=None):
 
     def write_config(
         self,
-        repo_version,
         wc_path=None,
         bare=False,
     ):
-        repo_version = int(repo_version)
-        if repo_version not in REPO_VERSIONS:
-            raise click.UsageError(f"Unknown sno repo version {repo_version}")
         # Bare-style sno repos are always implemented as bare git repos:
         if self.is_bare_style_sno_repo():
             self.config["core.bare"] = True
         # Force writing to reflogs:
         self.config["core.logAllRefUpdates"] = "always"
         # Write sno repo version to config:
-        self.config[SnoConfigKeys.SNO_REPOSITORY_VERSION] = str(repo_version)
+        self.config[SnoConfigKeys.SNO_REPOSITORY_VERSION] = str(LATEST_REPO_VERSION)
         # Write working copy config:
         WorkingCopy.write_config(self, wc_path, bare)
 
+    def ensure_latest_version(self):
+        if self.version != LATEST_REPO_VERSION:
+            raise InvalidOperation(
+                f"Sno repo uses Datasets v{self.version}, which is no longer supported.\n"
```

Comment:
```suggestion
                f"This repo uses Datasets v{self.version}, which is no longer supported.\n"
```

Reply:
Done: Sno repo -> This Sno repo.

Just like the output from `sno data version` this output is explicit that even though repos might be "Datasets V1" or "Datasets V2", they are still Sno repos. Seems good to me to keep the word "Sno" in the output.